nautobot/dcim/migrations/0066_controllermanageddevicegroup_radio_profiles_and_more.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 4.2.16 on 2024-11-05 13:48
```

Comment:
Does this have to be a separate migration from 0065?

Reply:
That's how Django generated the migrations. I am not sure why it is separate. I deleted all migrations and re-ran makemigrations.

Reply:
On reread, probably because of the inter-app dependency on these M2M. Fine as is.